lib/WP_Auth0_Nonce_Handler.php

```diff
@@ -166,13 +179,94 @@ public function generate_unique( $bytes = 32 ) {
 	 * @return bool
 	 */
 	protected function handle_cookie( $cookie_name, $cookie_value, $cookie_exp ) {
+		$illegal_chars = ",; \t\r\n\013\014";
+		if ( strpbrk( $cookie_name, $illegal_chars ) != null ) {
```

Comment:
This check just clues the administrator into an issue via the plugin error log if, for some reason, they're doing something wrong with customer state/nonce values. With one of these characters in place, the validation would be silently failing so this is not a breaking change.